packages/wasm/src/services/credential/service.ts

```diff
@@ -199,23 +199,19 @@ class CredentialService {
   }) {
     const searchParams = new URL(uri).searchParams;
     const params = new URLSearchParams(searchParams);
-    const credentialOfferEncoded = params.get('credential_offer');
-    const credentialOfferDecoded = decodeURIComponent(credentialOfferEncoded);
-    const credentialOffer = JSON.parse(credentialOfferDecoded);
-    const scope = credentialOffer.credentials[0];
-    const format = 'ldp_vc';
-    const credentialTypes = scope.replace('ldp_vc:', '');
 
     const client = await OpenID4VCIClientV1_0_13.fromURI({
       uri: uri,
       clientId: 'dock.wallet',
       authorizationRequest: {
         redirectUri: 'dock-wallet://credentials/callback',
         clientId: 'dock.wallet',
-        scope: credentialOffer.credentials[0],
       },
     });
 
+    const format = 'ldp_vc';
+    const credentialTypes = client.getCredentialsSupported()[0].scope.replace(`${format}:`, '');
```

Comment:
would be better to use `client.getCredentialsSupported()[0].scope.split(':')[1]` or similar, but also keep in mind that the scope parameter for 3d party implementations may have different formatting. so `ldp_vc:` is not guaranteed to exist. scope could just be like `AnotherCredentialType` or `sdjwt:AnotherCredentialType`

probably more robust to do:
```
const { scope }  = client.getCredentialsSupported()[0];
const scopeSplit = scope.split(':');
const credentialType = scopeSplit[scopeSplit.length - 1];
```